cmd/container.go

```diff
@@ -60,6 +60,8 @@ func newContainer(ctx context.Context, cfg config.ServerConfig, watchApps bool)
 
 			go ctr.ApplicationWatcher.Run(ctx, 1)
 		}
+	} else {
+		log.Debug().Msgf("not monitoring applications, MonitorAllApplications: %+v", cfg.MonitorAllApplications)
```

Comment:
using `Info` here might be useful, to make it clear to users what is happening and (maybe) why their applications aren't being monitored the way they expect

Reply:
changed to info